contentcuration/contentcuration/frontend/channelEdit/views/progress/ProgressBar.vue

```diff
@@ -8,14 +8,19 @@
       <Icon v-else-if="isDone" large color="greenSuccess" data-test="success">
         check_circle
       </Icon>
-      <VProgressCircular v-else indeterminate color="primary" />
+      <VProgressCircular
```

Comment:
I have removed this (should be in develop)

Reply:
That might explain the merge conflict then!